src/Uno.UI/UI/Xaml/DependencyProperty.Dictionary.NameToProperty.cs

```diff
@@ -26,11 +27,11 @@ public sealed partial class DependencyProperty
 	{
 		private class NameToPropertyDictionary
```

Comment:
This dictionary has a single static instance that is kept for the lifetime of the whole app.